src/components/Notification/InactivityMessage.jsx

```diff
@@ -91,7 +106,7 @@ const InactivityMessage = () => {
               variant="outlined"
               color="error"
               endIcon={<LogoutIcon />}
-              onClick={handleLogout}
+              onClick={() => logout()}
```

Comment:
Sorry Brandon, I should have been more clear in my last comment. The log out button for this component should contain the same call for localStorage.clear() as the useEffect.

I'll approve for now, since everything else looks good. Thanks for the updates.

Reply:
Oof, sorry I should've thought about that. I've added it now, thanks for approving